src/firefly/java/edu/caltech/ipac/firefly/data/ServerParams.java

```diff
@@ -58,6 +58,7 @@
     public static final String ID = "id";
     public static final String BID = "bid";
     public static final String CHANNEL_ID = "channelID";
+    public static final String TRY_MS = "tryMS";
```

Comment:
maybe naming it something like CONN_TIMEOUT or WAIT_TIME will have some implicit meaning.